Dockerfile.test

```diff
@@ -0,0 +1,12 @@
+FROM ruby:2.2.4
```

Comment:
Newer Ruby! 👍 

Glad to see it. Are there any potential regressions/odd behavior that you think this base image switch might bring up?